tui/network.py

```diff
@@ -159,6 +159,44 @@ def iface_details(context, conf):
     tui.screen.popHelpLine()
     return True
 
+def lacp_bond_ui(conf):
```

Comment:
Next time you test it, could you take screenshots of the UI?

Reply:
ok, when it's ready enough ;)